flipperdevices/flipperzero-firmware
# [FL-2872] Remove unused resources

# What's new

- Remove unused icons
- Replace icon in the "Restore Card Data" NFC scene with a slightly different one

# Verification 

- Check that everything compiles properly with no errors
- Try restoring a card dump to original, check that the arrows in the icon have no shadows

# Checklist (For Reviewer)
